website/docs/d/is_volume.html.markdown

```diff
@@ -33,6 +33,7 @@ Review the argument references that you can specify for your data source.
 ## Attribute reference
 In addition to all argument reference list, you can access the following attribute references after your data source is created.
 
+- `bandwidth` - The maximum bandwidth (in megabits per second) for the volume
```

Comment:
Add the type of aguement/attribute just like all other attributes in all doc files